src/estimation/src/BerdyHelper.cpp

```diff
@@ -2076,6 +2169,8 @@ bool BerdyHelper::serializeSensorVariables(SensorsMeasurements& sensMeas,
         setSubVector(y,sensorRange,toEigen(linkJointWrenches(childLink)));
     }
 
+    // TODO: Handle CoM acceleration sensor
```

Comment:
To be removed?

Reply:
Done in https://github.com/robotology/idyntree/pull/571/commits/f57ab3c061b91f83c5351d671132cc922ca75556

Reply:
Not exactly, if I understood well `serializeSensorVariables` method can be called to do an explicit initialization of the measurement vector from various measurement variables. I updated the method to consider to measurements from COM Accelerometer sensors https://github.com/robotology/idyntree/pull/571/commits/23951a2561aa2f2cef6b8d3486b5ec77f96cd4f8